webdriver/src/com/googlecode/jmeter/plugins/webdriver/sampler/gui/WebDriverSamplerGui.java

```diff
@@ -139,15 +134,20 @@ public void actionPerformed(ActionEvent actionEvent) {
 
     private void setScriptContentType(String ctype) {
         String text = script.getText();
-        script.setContentType(ctype);
+        script.setSyntaxEditingStyle(ctype);
         script.setText(text);
     }
 
     private JPanel createScriptPanel() {
-        script = new JEditorPane();
-        final JScrollPane scrollPane = new JScrollPane(script);
-        setScriptContentType("text/plain");
-        script.setFont(new Font(Font.MONOSPACED, Font.PLAIN, 14));
+    	script = new JSyntaxTextArea(20, 20, false);
+    	script.getFoldManager().setCodeFoldingEnabled(true);
+    	script.setFont(new Font("Hack", Font.PLAIN, 14));
```

Comment:
I'd just use Font.MONOSPACED constant. People are so subjective on fonts beauty, you know...  Let JVM settings to handle this. Using Jmeter property is an overkill IMAO.